### DISCUSSION TITLE
This password reset token is invalid

### DISCUSSION THREAD

**Discussion by adam-jones-net** [area17/twill#1647] [Q&A]
I don't often need to create new user accounts on our admin but when i do I often get told by the recipient that they can't define their password, receiving this error when they follow the given link and enter their details:

![image](https://user-images.githubusercontent.com/3810108/168795560-6a00f604-7655-4553-afcf-f132f73359f0.png)

Is this simply because they followed the link too late ?   If I want to manually now fix this, what do I need to add/edit in the database to make it work ?

**Comment by ifox** [comment]
That's correct @arkid, the current default is set to expire within 1 hour. https://github.com/area17/twill/blob/b32623b59c5bee4094f072419fed33067698cbe6/src/TwillServiceProvider.php#L201

**Comment by ifox** [comment]
Users can use the reset password functionality to get a new link in those cases.

In Twill 3, admins can resend the email, manually set a password, etc... which should help with those issues.
